### ISSUE TITLE
VGSEditText input type

### ISSUE THREAD

**Issue by boroslorena** [verygoodsecurity/vgs-collect-android#219]
I'm using `com.verygoodsecurity.vgscollect.widget.VGSEditText` UI component for inserting ZIP code information, when adding new credit card. I would like to limit the user so he can insert only digits in that field.
Input type can be set with  `app:inputType="number"` in xml layout, as per documentation... https://www.verygoodsecurity.com/docs/vgs-collect/android-sdk/vgs-edit-text#specify-the-keyboard-type

...but, it will always get overriden  by `applyInputType() `in InfoInputField class.
```
 private fun applyInputType() {
        val type = inputType
        if(type == InputType.TYPE_TEXT_VARIATION_PASSWORD || type == InputType.TYPE_NUMBER_VARIATION_PASSWORD) {
            inputType = InputType.TYPE_CLASS_TEXT or InputType.TYPE_TEXT_VARIATION_PASSWORD
        } else {
            inputType = InputType.TYPE_CLASS_TEXT
        }
        refreshInput()
    }
``` 
so, inputType can never be InputType.TYPE_CLASS_NUMBER.

So, is there a way how we can access VGSEditText.inputField and programmaticaly set inputType? 
Currently, its private and non-accessible. 

Also, there seems to be an issue with `com.verygoodsecurity.vgscollect.widget.PersonNameEditText`. I would like to have a capital first letter for person's first name and last name. But there is no method which we could use.

**SDK Version [1.6.6]**

**Comment by DmytroDm** [comment]
@boroslorena Hi, thank you for your information.

To fix `VGSEditText` please update VGS Collect SDK to at least version 1.6.8 or newer.

About `PersonNameEditText`, we can add inputType `textCapWords` ([android doc](https://developer.android.com/reference/android/text/InputType#TYPE_TEXT_FLAG_CAP_WORDS)), is this what are you looking for?

**Comment by Endorf** [comment]
Hi @boroslorena.
Today we published a new release of our SDK `v1.6.12`.
We fixed `VGSEditText` behavior. Now you can use `number` input type with `VGSEditText`.
Also, we added new `textCapWords` input type, so you could use it with `VGSEditText` and `PersonNameEditText`. All input types can be configured in code and from XML as well.
